src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/LocalAppContextSwitches.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    internal static partial class LocalAppContextSwitches
```

Comment:
Honestly, I wouldn't use `LocalAppContextSwitches`. Instead I would follow the simple pattern, like what we use in Resources:

[https://github.com/dotnet/runtime/blob/7ab65423373f04998f3d365d564b3fc0e55fdf2b/src/libraries/System.Private.CoreLib/src/System/Resources/ResourceReader.cs#L131](https://github.com/dotnet/runtime/blob/7ab65423373f04998f3d365d564b3fc0e55fdf2b/src/libraries/System.Private.CoreLib/src/System/Resources/ResourceReader.cs#L131)